auto_submit/lib/service/github_service.dart

```diff
@@ -147,17 +164,43 @@ class GithubService {
     );
   }
 
+  /// Get a particular comment by its id.
+  Future<IssueComment> getComment(
+    RepositorySlug slug,
+    int commentId,
+  ) async {
+    return await github.issues.getComment(
+      slug,
+      commentId,
+    );
+  }
+
+  /// List all the comments on a particular issue.
+  Future<List<IssueComment>> listCommentsByIssue(
+    RepositorySlug slug,
+    int issueNumber,
+  ) async {
+    return await github.issues
+        .listCommentsByIssue(
+          slug,
+          issueNumber,
+        )
+        .toList();
+  }
+
   /// Automerges a given pull request with HEAD to ensure the commit is not in conflicting state.
-  Future<void> autoMergeBranch(PullRequest pullRequest) async {
+  Future<bool?> autoMergeBranch(PullRequest pullRequest) async {
```

Comment:
Please document the return values, specially what null return code means.

Reply:
Done!